src/UIAutomation/FunctionalTests/TestUtils.h

```diff
@@ -18,3 +48,50 @@ void LogOutput(Args&&... args)
     Logger::WriteMessage(res.c_str());
 }
 
+// Waits until the given element gains focus.
+// Fails if the element doesn't gain focus after the given number of retries.
+// Returns the found element on success.
+inline winrt::com_ptr<IUIAutomationElement> WaitForElementFocus(const wchar_t* elementName, int retries = 100)
+{
+    static constexpr DWORD c_sleepMilliseconds = 50;
+    winrt::com_ptr<IUIAutomation> automation;
+    THROW_IF_FAILED(::CoCreateInstance(__uuidof(CUIAutomation8), nullptr, CLSCTX_INPROC_SERVER, IID_PPV_ARGS(automation.put())));
+
+    for (int i = 0; i < retries; ++i)
+    {
+        winrt::com_ptr<IUIAutomationElement> element;
+        const auto getFocusedElementHr = automation->GetFocusedElement(element.put());
+        if (getFocusedElementHr == E_INVALIDARG)

```

Comment:
When is this true?

Reply:
This is copied from the deleted FunctionalTests.cpp, which I assume is copied from CommonUtil.cpp, which has a comment explaining this. @mlalic, is there a reason you removed the comment?

Reply:
Resolved offline.